test/torch_np/test_basic.py

```diff
@@ -571,6 +572,7 @@ def test_ndarrays_to_tensors():
     assert isinstance(out[0][0], torch.Tensor)
 
 
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="requires cuda")
```

Comment:
There's a `skipIfNoCuda` (not real name, but you know) wrapper we use for this.

Reply:
https://github.com/pytorch/pytorch/blob/main/test/dynamo/test_misc.py#L2391 uses `skipif(not TEST_CUDA)`, and `TEST_CUDA` is precisely the same: https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_utils.py#L1062. Will change.